packages/wxt/src/core/create-server.ts

```diff
@@ -111,13 +111,15 @@ async function createServerInternal(): Promise<WxtDevServer> {
       await buildAndOpenBrowser();
 
       // Listen for file changes and reload different parts of the extension accordingly
-      const reloadOnChange = createFileReloader(server);
+      const reloadOnChange = () => {
+        keyboardShortcuts.start();
+        keyboardShortcuts.printHelp({
+          canReopenBrowser:
+            !wxt.config.runnerConfig.config.disabled && !!runner.canOpen?.(),
+        });
+        return createFileReloader(server);
+      };
       server.watcher.on('all', reloadOnChange);
```

Comment:
I think this line is still wrong...

```suggestion
      const reloadOnChange = createFileReloader(server);
      server.watcher.on('all', async (...args) => {
        await reloadOnChange(...args);
        
        // Restart keyboard shortcuts after file is changed - for some reason they stop working.
        keyboardShortcuts.start();
      });
      
      keyboardShortcuts.printHelp({
        canReopenBrowser:
          !wxt.config.runnerConfig.config.disabled && !!runner.canOpen?.(),
      });
```

Reply:
I've updated the code as you recommended, I've passed the first two arguments as it was throwing type error.